_pages/application.md

```diff
@@ -3,97 +3,131 @@ layout: doc
 title: "Course Staff Application"
 ---
 
-## CS 241 Course Assistant Positions Available!
+Are you interested in becoming a course assistant for CS 241? If so, then you are welcome to apply for positions. Although we have specialized roles, there is always plenty of work to do and course staff often work on areas they did not start out doing.
+For now, please pick one secondary role, and up to 2 primary roles that sound most interesting to you, but do read through all the descriptions!
 
-Are you interested in becoming a course assistant for CS 241? If so, then you are welcome to apply for positions. Please pick at most two areas that interest you, but read through all the descriptions!
+## Roles
 
-* Student-Facing (Labs, Office Hours)
-* Honors Section
-* Assignment Development
-* Infrastructure
+### Primary
+- Lab and Office hours
+- Assignment Development
+- Honors (CS 296-41)
+- Infrastructure
 
-All positions require dedicated people who have a passion for teaching system programming. Those who submit a promising application will be offered an interview. We also want to be completely transparent about what we look in our applicants including
+### Secondary
+- Data Analytics and Research
+- Web and Coursebook Development
 
-* Systems programming aptitude (not just an A+ in 241)
+All positions require dedicated people who have a passion for teaching system programming. Those who submit a promising application will be offered an interview. We also want to be completely transparent about what we look in our applicants. 
+We are not simply looking for A+ students. A good grade is neither necessary nor sufficient to be a good course assistant. Finally, we expect everyone on course staff to contribute regularly to Piazza, to help answer student questions. You _must_ have a primary role to be accepted, but secondary roles are optional.

```

Comment:
Accepted should be considered.

A bit more detail into what distinguishes a primary from a secondary role and why there is such a distinction would be nice.

My thoughts on reading this were to wonder whether it's really necessary to even mention the secondary roles in such a formal manner here. The fact that you require a primary seems to imply that these are related to ongoing projects on staff, but will a candidate who applies for a secondary role be perceived as better than an equivalent candidate who didn't apply for a secondary role?

Reply:
> Accepted should be considered
Sounds good.

> will a candidate who applies for a secondary role be perceived as better than an equivalent candidate who didn't apply for a secondary role?
Possibly, if they have a lot of experience and/or show more commitment.

The idea behind adding secondary roles was to take some of these "extra" responsibilities and move them to another role. This should hopefully free up staff to respond more on Piazza, focus and deliver on a few key responsibilities, and so we have bandwidth to do more in AD / Infrastructure / Analytics.